ci/docker/runtime_functions.sh

```diff
@@ -1014,8 +1001,10 @@ unittest_ubuntu_python3_cpu_mkldnn() {
     export MXNET_SUBGRAPH_VERBOSE=0
     export MXNET_ENABLE_CYTHON=0
     export DMLC_LOG_STACK_TRACE_DEPTH=10
-    # TODO(szha): enable parallel testing and naive engine for ops once #18244 is fixed
-    pytest --durations=50 --cov-report xml:tests_unittest.xml --verbose tests/python/unittest
+    pytest -m 'not serial' -k 'not test_operator' -n 4 --durations=50 --cov-report xml:tests_unittest.xml --verbose tests/python/unittest
```

Comment:
It would be better to set OMP_NUM_THREADS=total_cpu_threads/4. Otherwise multiprocessing module from python will spawn 4 processes which will be using all threads independently (4 x total_cpu_threads).
For serial you can then unset OMP_NUM_THREADS